### ISSUE TITLE
(Question) Request for a supported product matrix

### ISSUE THREAD

**Issue by apgsga-jhe** [eclipse/birt#667]
Hi guys,

I've been using BIRT since a long time (even working for Actuate, before it went bought by OpenText), and i'm happy to see that a "new community" is getting built around the OpenSource Version again.

We were using the Plugin version 4.7.0.v201706222054 with Eclipse 2019-09 (4.13.0) and we had problem to open existing rptdesign. After having updated the plugin to the last version you guys built (4.9.0.v202108150822), all went fine, cheers!!

Our rptdesign are embedded into a Java Application which uses the Birt Runtime Engine to render reports as PDF. In order to get the engine, our application has the following dependency:

```
 <dependency>
	<groupId>org.eclipse.birt.runtime</groupId>
	<artifactId>org.eclipse.birt.runtime</artifactId>
	<version>4.4.2</version>
</dependency>
```

Looking at the central Maven Repository, it seems that nothing's really moving with BIRT Runtime: https://search.maven.org/classic/#search%7Cgav%7C1%7Cg%3A%22org.eclipse.birt.runtime%22%20AND%20a%3A%22org.eclipse.birt.runtime%22

Do you have somewhere a kind of supported product matrix which would provide the following information:

- Which Eclipse Version is supported by which plugin
- Which BIRT Runtime Version supports which rptdesign
- Which Java Version is supported by which BIRT Runtime

Thanks!!

**Comment by wimjongman** [comment]
Thanks for filing the issue. The 4.9 version has not been released. We are still fighting some tough issues. After we have released it, we can create such a matrix.

**Comment by apgsga-jhe** [comment]
Cheers, looking forward to it.

**Comment by apgsga-jhe** [comment]
@wimjongman : one additional quick question ... do you already (more or less) know when 4.9.0 will officially be released?

**Comment by wimjongman** [comment]
> 
> 
> @wimjongman : one additional quick question ... do you already (more or less) know when 4.9.0 will officially be released?

I expect it to be in December.

**Comment by wimjongman** [comment]
Closing since we released 4.9
